pytest_pootle/fixtures/views.py

```diff
@@ -134,21 +134,18 @@ def project_views(request, client):
     return test_type, project, response.wsgi_request, response, kwargs
 
 
-@pytest.fixture(params=TP_VIEW_TESTS.keys())
-def tp_view_test_names(request):
-    return request.param
-
-
 @pytest.fixture(params=(None, 'member', 'admin'))
-def tp_view_usernames(request):
+def username(request):
```

Comment:
yep - but if you make the fixture singular its not clear that the test is parametrized

my point about ``usernames`` is that im going to change it to include passwords - ie dict not string - but i can happily rename it when i do